### PR TITLE
User can list a space test, erb, routes

### PR DESCRIPTION
**Pull request by AlexandraGF**
Co-authored-by: Reena <reena.sanjay.sharma@gmail.com>

### CODE REVIEW

File: app/app.rb [hunk 1]
```diff
@@ -1,30 +1,51 @@
...
+    session[:space_name] = params[:space_name]
+    session[:space_description] = params[:space_description]
+    session[:space_price] = params[:space_price]
+    session[:space_availability] = params[:space_availability]
+    # redirect '/new_space'
+    redirect '/spaces'
+    # erb :new_space
+  end
+
+  # get '/new_space' do
```
Comment (bpourian) [inline app/app.rb]: make sure you delete unused code 👎

File: app/app.rb [hunk 2]
```diff
@@ -1,30 +1,51 @@
...
+    @space_price = session[:space_price]
+    @space_availability = session[:space_availability]
+    erb :spaces
   end
 
+  get '/new_space' do
+    erb :new_space
+  end
+
+  post '/new_space' do
```
Comment (bpourian) [inline app/app.rb]: I think we should change the route to '/spaces/new'

**Review by bpourian — CHANGES_REQUESTED** [review]
change routes name as per comments and do another pull please

**Review by bpourian — APPROVED** [review]

**Comment by bpourian** [comment]
🦑 

Merge conflict resolved in group

**Comment by samuelmaker** [comment]
💯

**Comment by reenz** [comment]
👍

**Pull request merged**